### PR TITLE
build: auto release functionality

### PR DESCRIPTION
**Pull request by malikovmm**

### CODE REVIEW

**Comment by malikovmm** [comment]
works the same as in scan2html. You need to change the version field in plugin.yaml

**Comment by afdesk** [comment]
@malikovmm please, update this PR

File: src/template.go
Comment (afdesk) [inline src/template.go:1]: this is old file

**Review by afdesk — CHANGES_REQUESTED** [review]

**Review by afdesk — APPROVED** [review]

**Comment by afdesk** [comment]
thanks

**Pull request merged**